### DISCUSSION TITLE
Windows subversion not updating. (KB4566116)

### DISCUSSION THREAD

**Discussion by NamanJain1902** [microsoft/WSL#5882] [General]
I want to update Windows 10 Version 1909 for x64 based System(KB4566116).
Currently it's 1909(OS Build 18363.1016).

I have downloaded 2020-08 Cumulative Update Preview for Windows 10 Version 1909 for x64 based System(KB4566116 ) from Microsoft Update Manager.
After installation and rebooting the PC, system is unable to update it. 

Windows Version still reflecting the same version.

**Comment by therealkenc** [comment]
Ref #5790

**Reply by NamanJain1902** [reply]
@therealkenc  That thread didn't solve my issue. I have downloaded file to upgrade subversion from 1016 to 1049 but it's not updating.
